js/modals.js

```diff
@@ -0,0 +1,85 @@
+
+
+const closePopUp = document.getElementsByClassName('popup_close');
```

Comment:
исправить на акутальный стандарт es6+
```suggestion
const closePopUp = document.querySelector('.popup_close');
```